x-pack/test/plugin_api_integration/test_suites/event_log/public_api_integration.ts

```diff
@@ -18,137 +18,162 @@ export default function ({ getService }: FtrProviderContext) {
   const supertest = getService('supertest');
   const log = getService('log');
   const retry = getService('retry');
+  const spacesService = getService('spaces');
+  const esArchiver = getService('esArchiver');
 
   describe('Event Log public API', () => {
-    it('should allow querying for events by Saved Object', async () => {
-      const id = uuid.v4();
+    before(async () => {
+      await spacesService.create({
+        id: 'namespace-a',
+        name: 'Space A',
+        disabledFeatures: [],
+      });
+    });
 
-      const expectedEvents = [fakeEvent(id), fakeEvent(id)];
+    after(async () => {
+      await esArchiver.unload('empty_kibana');
+    });
 
-      await logTestEvent(id, expectedEvents[0]);
-      await logTestEvent(id, expectedEvents[1]);
+    for (const namespace of [undefined, 'namespace-a']) {
```

Comment:
The tests here previously just tested an event with a SO in the default space, so I refactored all those tests to test with the default space AND a named space.  Hence the indentation changes, and parameterization of the namespace.  The diff came out much worse than I expected :-)